charts/rancher-monitoring/100.2.0+up40.1.2/app-README.md

```diff
@@ -14,6 +14,20 @@ The chart installs the following components:
 
 For more information, review the Helm README of this chart.
 
+## Upgrading from 100.1.0+up19.0.3 to 100.2.0+up40.1.2
+
+### Rebased the chart to upstream version 40.1.2. 
+### Noticeable changes:
+
+Grafana: 
+- `grafana.serviceMonitor.enabled` default is set to `true`
+
+Prometheus-Operator:
+- Introduced a change to avoid name collisions with AlertmanagerConfig (https://github.com/prometheus-operator/prometheus-operator/commit/80a72c42f5c62012d27330b7c5203fd110d7f764)
+
+- Reverted default metrics relabelings for cAdvisor added in 36.x. If you do not want this change, you will need to override the `kubeApiServer.metricRelabelings` and/or `kubelet.cAdvisorMetricRelabelings`
```

Comment:
I think this can be omitted, unless there’s an end user impact in terms of how they would interact with Monitoring as a result?